source/change-streams/change-streams.rst

```diff
@@ -544,8 +525,34 @@ Option 1: ChangeStream::getResumeToken()
     public getResumeToken() Optional<Document>;
   }
 
-This method returns the cached ``documentResumeToken`` or ``postBatchResumeToken`` following rules from `Resume Process`_:
+This method returns the cached ``documentResumeToken`` or ``postBatchResumeToken`` following these rules:
+
+- If the ``ChangeStream`` has a cached ``postBatchResumeToken`` and has returned all documents in the most recent batch (or the most recent batch was empty), return the cached ``postBatchResumeToken`` and:
+
+  - The driver MUST set ``resumeAfter`` to the cached ``postBatchResumeToken``.
+  - The driver MUST NOT set ``startAfter``. If ``startAfter`` was in the original aggregation command, the driver MUST remove it.
+  - The driver MUST NOT set ``startAtOperationTime``. If ``startAtOperationTime`` was in the original aggregation command, the driver MUST remove it.
```

Comment:
The "MUST set" and "MUST NOT set" lines here are no longer relevant, since we're only specifying what is returned. You can probably delete the trailing "and:" from each top-level bullet (e.g. "if" and "else if" conditions) and just remove the sub-bullets entirely.

You may want to preserve one set of the MUST and MUST NOT bullets to use above in the resume process docs. If `getResumeToken()` actually returns a value, I believe you'd always want to set it as `resumeAfter` and never set `startAfter` and `startAtOperationTime`.